docs/basics/basic_scenario.md

```diff
@@ -0,0 +1,49 @@
+# Scenario
+
+Here is the basic scenario structure.  Scenario can be one or many.  When many scenarios are specified, they will be executed sequentially.  For more detail, please see [scenario]({{ "/docs/concepts.html#scenario" | absolute_url }})
```

Comment:
> Scenario can be one or many.

There's always single *scenario*. Scenario consists of *sequences*, and these consists of *steps*.

> they will be executed sequentially.

Sequences won't be executed serially. Steps in each instance of sequence will be executed serially.

Reply:
I have reworded.  please have a look.